### PR TITLE
python311Packages.pygame: marked broken for python >3.11

### PR DESCRIPTION
**Pull request by superherointj**
python311Packages.pygame: mark broken for python >3.11

> src_c/pypm.c:209:12: fatal error: longintrepr.h: No such file or directory

* It is already broken in master.
* `python310Packages.pygame` builds fine. Seems some issue specific to 3.11. (But don't ask me!)

Logs: https://termbin.com/l71yq

Only marking broken to reduce false positives in downstream builds.

### CODE REVIEW

File: pkgs/development/python-modules/pygame/default.nix [hunk 1]
```diff
@@ -75,5 +76,7 @@ buildPythonPackage rec {
     license = licenses.lgpl21Plus;
     maintainers = with maintainers; [ emilytrau ];
     platforms = platforms.unix;
+    # fatal error: longintrepr.h: No such file or directory.
+    broken = pythonAtLeast "3.11"; # At 2022-02-27
```
Comment (wegank) [inline pkgs/development/python-modules/pygame/default.nix:80]: How about `disabled = pythonAtLeast "3.11"`?

**Comment by superherointj** [comment]
Result of `nixpkgs-review pr 218680` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)

File: pkgs/development/python-modules/pygame/default.nix (hunk 1 above)
Comment (superherointj) [reply pkgs/development/python-modules/pygame/default.nix]: `nixpkgs-review` is not showing it as marked broken. Idk, are you sure this is better?

Comment (superherointj) [reply pkgs/development/python-modules/pygame/default.nix]: I would rather have nixpkgs-review show it as `marked broken` instead of silent failure.

**Comment by superherointj** [comment]
Result of `nixpkgs-review pr 218680` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)

**Comment by superherointj** [comment]
Ok, same result. I'm not sure what is the difference.
From what I understand, the package won't build because it is broken (for whatever reason). So marking broken does make sense.

File: pkgs/development/python-modules/pygame/default.nix (hunk 1 above)
Comment (superherointj) [reply pkgs/development/python-modules/pygame/default.nix:80]: I have reversed change to `disabled`. (So I don't know if you mark if conversation is resolved is up to date.)

**Comment by superherointj** [comment]
@wegank Thanks!

**Pull request merged**